lib/wasi/Cargo.toml

```
@@ -19,6 +19,7 @@ log = "0.4.6"
 byteorder = "1.3.1"
 # hack to get tests to work
 wasmer-singlepass-backend = { path = "../singlepass-backend", version = "0.4.2", optional = true }
+winapi = "0.3"
```

I think we can depend on this only on windows:

```suggestion

[target.'cfg(windows)'.dependencies]
winapi = "0.3"
```